tags/TagSet.md

```diff
@@ -0,0 +1,109 @@
+# Summary
+A `Tag` is used to label anything that is associated
+with a specific operation, such as an HTTP request. These `Tag`s are used to
+aggregate measurements in a [`View`](https://github.com/census-instrumentation/opencensus-specs/blob/master/stats/DataAggregation.md#view) 
+according to unique value of the `Tag`s. The `Tag`s can also be used to filter (include/exclude)
+measurements in a `View`. `Tag`s can further be used for logging and tracing.
+
+# Tag
+A `Tag` consists of TagScope, TagKey, and TagValue.
+
+## TagKey
+
+`TagKey` is the name of the Tag. TagKey along with `TagValue` is used to aggregate
+and group stats, annotate traces and logs.
+
+**Restrictions**
+- Must contain only printable ASCII (codes between 32 and 126 inclusive with below Exception)
+- - Exceptions '=', ','
+- Must have length greater than zero and less than 256 combined with the source and metadata.
+- Must not be empty.
+
+## TagValue
+
+`TagValue` is a string or string wrapper. It MUST contain only printable ASCII (codes between
+32 and 126)
+
+## TagScope
+
+`TagScope` is used to determine the scope of a `Tag`. For now, it has two possible values,
+Local or Non-Local. In future, additional values be added to address specific situations.
```

Comment:
Maybe local and global?

Reply:
`Scope.(Process | Local)`, `Scope.(Request | Transaction | Execution)`.

`global` sounds a bit scary & imprecise. `remote` hints at origin rather than scope.

Reply:
| is "or", so values:
* local: Process or Local
* global: Request or Transaction or Execution

Reply:
so you are suggesting ScopeType (local | gloabl) and ScopeValues (according to Scope type). Since scope is not propagated how will these values be used? Will they be used by backend?

Reply:
I was just suggesting alternative names for local/global

Reply:
I agree global is not that good. Probably Request or Transaction are better names.